package.json

```diff
@@ -41,7 +41,8 @@
     "reconnecting-websocket": "4.4.0",
     "uniqid": "5.4.0",
     "ws": "8.4.0",
-    "xhr2": "0.2.1"
+    "xhr2": "0.2.1",
+    "@khavryliuk/json-bigint": " 1.0.0"
```

Comment:
What do you mean repos? It's an NPM package.

Reply:
This one points to mine NPM namespace, coz seems like mlabs does not have one, yet. And as we use npm2nix we can not install node packages from github.

Reply:
@kirill-havryliuk we do have an MLabs NPM namespace. I've messaged you on slack to invite you to it.

Reply:
How does our fork differ compared to the one used by ogmios, for example?

Reply:
They use multiple versions of json-bigint for some reason. 
    "@cardanosolutions/json-bigint": "^1.0.0",
    "@types/json-bigint": "^1.0.1"
    
  @cardanosulutions/json-bigint is buggy, it can round numbers sometimes and loss precision and in some cases. 
  
  Our version is fixed: all deciding logic, about how to parse numbers was stripped. It always parse all numbers as BigNumber without any assumptions.

Reply:
>  "@types/json-bigint": "^1.0.1"

That's just for the type annotations for typescript, I believe

I was under the impression that the one by cardanosolutions had the correct behaviour, at least for scientific notation numbers, given it's used by the client library for ogmios itself. Is there some behaviour with floating point numbers?

Reply:
The only difference: https://github.com/CardanoSolutions/json-bigint/commit/d277387d7750ff4e698677d0aa0db97ebb60c581

And they introduced a bug with this change. 
You can try it yourself: `require("@cardanosolutions/json-bigint").parse("10000000000000.001")`